src/main/java/com/gmail/inverseconduit/chat/StackExchangeChat.java

```diff
@@ -36,10 +36,11 @@
 
     private final Set<ChatWorker>                 subscribers            = new HashSet<>();
 
-    private final Set<Long>                       handledMessages        = new HashSet<>();
+    private final Set<Integer>                       handledMessages        = new HashSet<>();
```

Comment:
ya there are some (IMO strange) formatting conventions for the repo that require aligning field declarations for reasons ...

There's an eclipse formatting preferences file at the root of the repository named `'prefs.epf'`. 
Please either manually align the declarations or make eclipse (eurgh) do it for you. Thanks :)

(note that this is a nitpick and I do like the change)